### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by adelphinsucasa**
Add Popoup Windows to my project
- Desktop and mobile popup windows using javascript and css, when the viewer click or tap on a project button, the popup window is displayed.
- The Desktop image is different to the mobile image in the popup window

### CODE REVIEW

**Review by ishimwezachee — COMMENTED** [review]
Hi adelphinsucasa  and chelobotix 
While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
image

## Invalid Code Review Request

You have submitted a project, whose style linters are not passing. please fix it and re-submit it for the review. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: popup.js [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+const ulModal = document.querySelector('ul.modal-project');
+
+function createModal(index) {
+  const imgModal = document.querySelector('img.modal-project');
+  const h3Modal = document.querySelector('h3.modal-project');
+  const btn1Modal = document.querySelectorAll('.modal-btn1');
+  const btn2Modal = document.querySelectorAll('.modal-btn2');
+  const pModal = document.querySelector('p.modal-project');
+
+  if (window.matchMedia('(max-width: 767px)').matches) {
```
Comment (christianbiring1) [inline popup.js:133]: [OPTIONAL]
- [ ] You may consider readjusting the background color of your `see project` button on the movie version
image

**Review by christianbiring1 — APPROVED** [review]
Hi @adelphinsucasa , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 
APPROVED

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`
- [ ] The project section is rendered dynamically :+1: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by adelphinsucasa** [comment]
Great Thank you very much for your time.

**Pull request merged**